libs/platform/src/lib/components/wizard-generator/components/dialog-wizard-generator/dialog-wizard-generator.component.scss

```diff
@@ -0,0 +1,46 @@
+.fdp-wizard-dialog {
+
+    .fd-message-strip {
+        margin-bottom: 12px !important;
+    }
+
+    .fd-dialog {
+        &__header {
+            .fd-bar {
+                box-shadow: none;
+            }
+        }
+    }
+
+    .fd-title {
+        &--h3 {
+            margin-bottom: 12px !important;
+        }
+
+        &--h2 {
+            padding: 16px 0 8px 16px;
```

Comment:
for paddings please check https://fundamental-styles.netlify.app/?path=/docs/layouts-paddings--all-round-padding